src/api-v3/paths/accounts/{address}/settings.spec.ts

```diff
@@ -1,11 +1,13 @@
 import request from 'supertest';
-import { mockApp, rippleApi } from "../../../../fixtures/mocks";
+import { mockApp, rippleApi, mockedDebuglog } from "../../../../fixtures/mocks";
 import sinon from 'sinon';
-// import getAccountInfoFixture from '../../../../fixtures/getAccountInfo.json';
+import { expect } from 'chai';
+import { capture } from 'ts-mockito';
+
 const path = '/v3/accounts/{address}/settings';
 
-describe.skip(path, () => {
-  it('GET - returns account settings', (done) => {
+describe(path, () => {
+  it.skip('GET - returns account settings', (done) => {
```

Comment:
I'd place a:
```
TODO(intelliot): Re-enable after [condition that these tests can be enabled] is done.
```

That way future engineers will know who has context and you'll remember what the blocker here was.